### ISSUE TITLE
failed to build - renew.go - undefined: syscall.Kill

### ISSUE THREAD

**Issue by richp10** [foomo/simplecert#8]
Line 99 of renew.go

I'm trying to build on windows - is this library linux only?

**Comment by dreadl0ck** [comment]
I wasn't aware syscall.Kill is not available on windows.

Propably a SIGCONT could be used to trigger the reload and implement the functionality via https://github.com/shirou/gopsutil cross platform, as SIGHUP is not available on windows.

I can take a look at that next week.

**Comment by richp10** [comment]
Thanks - I think SIGHUP is available but not syscal.Kill (see types_windows.go)

Maybe the following is equivalent: 

```
p, err := os.FindProcess(os.Getpid())
if err != nil {
	return err
}
p.Signal(syscall.SIGHUP)
```

As a quick hack I replaced the offending line with the above and the lib compiled fully.

**Comment by dreadl0ck** [comment]
Oh even better - fixed with 6afb1e7.

Will create a new release now.

Thanks for your contribution!
